cypress/e2e/tests/migration/applicationinventory/assessment/archetype_association.test.ts

```diff
@@ -72,6 +73,43 @@ describe(["@tier2"], "Tests related to application-archetype association ", () =
         click(commonView.sideDrawer.closeDrawer);
     });
 
+    it("Verify application review from multiple archetypes", function () {
+        // Automates MTA-420
+        archetypeList = createMultipleArchetypes(2, tags);
+
+        const appdata = {
+            name: data.getAppName(),
+            description: data.getDescription(),
+            tags: [tags[0].name, tags[1].name],
+            comment: data.getDescription(),
+        };
+
+        const application = new Application(appdata);
+        applicationList.push(application);
+        application.create();
+        cy.wait(2 * SEC);
+
+        archetypeList[0].perform_review("low");
+        archetypeList[1].perform_review("medium");
+
+        // Assert that 'Archetypes reviewed' is populated on app drawer after review inheritance
+        Application.open();
+        sidedrawerTab(application.name, "Details");
+        cy.get(commonView.sideDrawer.associatedArchetypes).contains("Archetypes reviewed");
+        cy.get("dt")
+            .contains("Archetypes reviewed")
+            .closest("div")
+            .within(() => {
+                cy.get("dd").then(($value) => {
+                    let text = $value.text();
+                    expect(text).to.equal(archetypeList[0].name + archetypeList[1].name);
```

Comment:
please use `const` instead of `let` untill you are not going to change value later